### DISCUSSION TITLE
Unable to create new apps on the cloud with "document update conflict" error

### DISCUSSION THREAD

**Discussion by dannyshin76** [Budibase/budibase#11243] [Help] [answered]
Today I tried to create a new app (from scratch) on the cloud, but failed with this console error:
![image](https://github.com/Budibase/budibase/assets/93232582/d0fa46ee-57b4-4493-887c-da33d3100385)

Also I tried to delete some previous apps but it weirdly fails too, creating bunch of "ghost" apps that I see but can't access.
Anyone having the same problem or is it only me? I remember I didn't have problem some time last week.

**Comment by dannyshin76** [comment]
All problems gone today. Maybe it was a temporary issue.

**Reply by ConorWebb96** [reply, marked as answer]
Hey @dannyshin76,

Are you using the cloud services? If so there was an issue yesterday that got resolved by the engineering team at around 10 am GMT time. So this may have indirectly fixed your issue.

Hopefully information makes sense.
